docs/getting_started.md

```diff
@@ -255,17 +255,19 @@ for i in range(100):
 '...and now we\'re done!'
 ```
 
-## Record a screencast
+## Share your app
 
-After you've built a Streamlit app, you may want to discuss some of it with co-workers over email or Slack, or share it with the world on Twitter. A great way to do that is with Streamlit's built-in screencast recorder. With it, you can record, narrate, stop, save, and share with a few clicks.
+After you’ve built a Streamlit app, it's time to share it! To show it off to the world you can use **Streamlit sharing** to deploy, manage, and share your app for free. Streamlit sharing is currently invitation only, so please [request an invite](https://www.streamlit.io/sharing) and we'll get you one soon!
 
-To start a screencast, locate the menu in the upper right corner of your app (**☰**), select **Record a screencast**, and follow the prompts. Before the recording starts, you'll see a countdown — this means it's showtime.
+<!-- [[we'll need to grab the gif after the video editors are done with it on Wednesday]] -->
```

Comment:
Sounds like we need to make an exception change tomorrow for this? Totally fine with me, just confirming what I should look out for

Reply:
Possibly, but I don't view this as a P0. No one will notice it coming in a week